regolith/tools.py

```diff
@@ -776,23 +775,24 @@ def is_fully_appointed(appts, begin=None, end=None, now=None):
         else:
             end_date = begin_date + relativedelta(years=1)
         timespan = end_date - begin_date
+        loading = []
+        max, min = 0.0, 1.0
         for x in range(0, timespan.days):
-            datearray.append(begin_date + relativedelta(days=x))
-        loading = [0.0] * len(datearray)
-        for x in range(0, len(datearray)):
+            loading.append(0.0)
+            day = begin_date + relativedelta(days=x)
             for appt in appts:
-                if is_current(appt, now=datearray[x]):
+                if is_current(appt, now=day):
                     loading[x] = loading[x] + appt.get("loading")
```

Comment:
I think appt["loading"] can be a list.  Will this work?

Reply:
this won't work in its current state, but I can change it to `loading[x] = loading[x] + sum(appt.get("loading"))` if you want that the total loading is the sum of the numbers in the list. could you please clarify the expected behaviour in the list case?

Reply:
please can you check the schema?  I may be wrong on this on second thoughts.

btw, please sync upstream and rebase before doing that as some recent merges have changed the schema quite a bit.

Reply:
I checked the schema on my synced `master` branch, `appointments` under `people`doesn't seem to have a subschema, and in the exemplar it is a float value – is there maybe a specific branch I should be looking at on your fork for a subschema for `appointments` ? also will do the rebasing!